src/views/SubjectAllQuestions/SubjectAllQuestions.tsx

```diff
@@ -11,6 +11,12 @@ import { useSubjectData } from 'hooks/useSubjectData/useSubjectData';
 import { Question } from './Question/Question';
 import { CreateExamModal } from './CreateExamModal/CreateExamModal';
 
+const formatQuestionsCountText = (count: number) => {
+  if (count === 1) return 'pytanie';
+  if (count >= 2 && count <= 4) return 'pytania';
+  return 'pytań';
+};
```

Comment:
Just a bit of a personal preference 😉 

```suggestion
const formatQuestionsCountText = (count: number) => {
  if (count === 1) {
    return 'pytanie';
  }

  if (count < 5) {
    return 'pytania'
  }

  return 'pytań';
};
```

Reply:
Okay, I'll add the brackets. However there is one thing regarding that second condition, it looks unnecessarily complicated, but it was actually done on purpose. That way passing `0` would yield `0 pytań`. Quite an edge case, but wanted to be correct.